src/package.xml

```diff
@@ -1192,6 +1192,7 @@
         <members>npe03__Recurring_Donation__c.Always_Use_Last_Day_Of_Month__c</members>
         <members>npe03__Recurring_Donation__c.Day_of_Month__c</members>
         <members>npe03__Recurring_Donations_Settings__c.Recurring_Donation_Batch_Size__c</members>
+        <members>npe03__Recurring_Donations_Settings__c.IsRecurringDonations2Enabled__c</members>
```

Comment:
Darn, I missed that this one is not in alphabetical order. One last change. Sorry.